opentelemetry/src/baggage.rs

```diff
@@ -282,10 +282,29 @@ impl FromIterator<KeyValueMetadata> for Baggage {
     }
 }
 
+fn encode(s: Cow<'_, str>) -> String {
```

Comment:
nit - can this function take `&str` as it is not modifying the input string?

Reply:
yep